ui/components/app/signature-request-siwe/index.scss

```diff
@@ -21,18 +21,25 @@
     box-shadow: 0 0 7px 0 rgba(0, 0, 0, 0.08);
   }
 
+  /** @todo replace ActionableMessage or remove overwritten code. */
```

Comment:
this is addressed in this follow-up PR: https://github.com/MetaMask/metamask-extension/pull/18207